satellite/metainfo/metainfo_test.go

```diff
@@ -435,6 +435,49 @@ func TestCommitSegmentPointer(t *testing.T) {
 	})
 }
 
+func TestValueAttributeInfo(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 6, UplinkCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		apiKey := planet.Uplinks[0].APIKey[planet.Satellites[0].ID()]
+
+		metainfo, err := planet.Uplinks[0].DialMetainfo(ctx, planet.Satellites[0], apiKey)
+		require.NoError(t, err)
+		projects := planet.Satellites[0].DB.Console().Projects()
+
+		project, err := projects.Insert(ctx, &console.Project{
+			Name:        "ProjectName",
+			Description: "projects description",
+		})
+		assert.NotNil(t, project)
+		assert.NoError(t, err)
+
+		keyInfo := pb.ValueAttributionRequest{
+			PartnerId: []byte("PartnerID"),
+			UserId:    []byte("userID"),
+			BucketId:  []byte("myBucketName"),
+		}
+
+		{
+			// error if pointer is nil
+			_, err = metainfo.CommitSegment(ctx, "bucket", "path", -1, nil, []*pb.OrderLimit2{})
```

Comment:
I'm not sure what those tests are doing because `CommitSegment` in both cases is trying to commit object not bucket. If I don't miss anything then we need tests for:
* assign value attribution - error (no bucket)
* create bucket - ok, assign value attribution - ok
* create bucket - ok, upload file - ok, assign value attribution - error
* create bucket - ok, assign value attribution partner A - ok, assign value attribution partner B - error

Reply:
added the suggested test case